content/250-postgres/300-database/550-local-development.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: 'Local development'
+metaTitle: 'Local development with Prisma Postgres'
+metaDescription: 'Guide to setting up local development using Prisma Postgres'
+tocDepth: 3
+toc: true
+---
+
+
+[Prisma Postgres](/postgres) is our production-grade, cloud-native database and is ideal for staging and production environments. For rapid iteration and isolated testing, you can run a local Prisma Postgres instance (powered by [PG Lite](https://pglite.dev)). This guide shows you how to install and launch a local Prisma Postgres database.
+
+
+## Setting up local development for Prisma Postgres
+
+Follow these steps to set up local Prisma Postgres for development.
+
+### 1. Launching local Prisma Postgres
+
+Navigate into your project and start the local Prisma Postgres server using the following command:
+
+```terminal
+npx prisma dev
+```
+
+This starts a local Prisma Postgres server that you can connect to using Prisma Client. The server will run in the background and will be available at `localhost:51213` by default and will also contain an auto-generated API key.
+
+This should give you an output with `DATABASE_URL` that looks like this:
```

Comment:
- "will run in the background" -> what does that mean? it doesn't really run in the background because the process is needs to stay active in the terminal, right? 
- "The server [...] and will also contain an auto-generated API key" -> does the server contain an API key or the URL? I think this entire paragraph can be rephrased to be more concise/clear

```suggestion
This starts a local Prisma Postgres server that you can connect to using Prisma ORM. 

The output of the command contains a `DATABASE_URL` and looks like this:
```